### DISCUSSION TITLE
How to personalize calendar keymap ?

### DISCUSSION THREAD

**Discussion by babakou9** [nvim-neorg/neorg#1154] [Q&A]
The default keymap for moving in the calendar does not follow user’s configuration and is 'hjkl' with no configuration option (that I could find).
For users with dvorak or bépo keyboards it isn’t intuitive to select a date. 
Is it possible to set a personalized keymap for this module ?

**Comment by pysan3** [comment]
https://github.com/nvim-neorg/neorg/blob/main/lua/neorg/modules/core/ui/calendar/views/monthly.lua#L584

Seems like there's literally a todo for that lol. 

I guess if you want this feature you need to send a feature request or implement on your own...

**Reply by max397574** [reply]
I think that todo is meant for sth else
but  yes it's not yet possible in a nice way

**Reply by babakou9** [reply]
thank you for your answers

**Reply by steevelefort** [reply]
Hi,
Same issue for me. It's really difficult to use on a non-qwerty based keyboard.

**Reply by pysan3** [reply]
This is currently hard coded so you should create a feature request by creating a new issue. There is no workaround whatsoever atm.
